tools/integration_tests/util/client/control_client.go

```diff
@@ -18,14 +18,45 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"reflect"
 	"strings"
+	"time"
 
 	control "cloud.google.com/go/storage/control/apiv2"
 	"cloud.google.com/go/storage/control/apiv2/controlpb"
+	"github.com/googleapis/gax-go/v2"
+	"google.golang.org/grpc/codes"
 )
 
+func storageControlClientRetryOptions(clientConfig *control.StorageControlClient) []gax.CallOption {
+	return []gax.CallOption{
```

Comment:
Shouldn't we try to reuse this https://github.com/GoogleCloudPlatform/gcsfuse/blob/21633e8ffc6f4d2c1c72e79e08996a98543c95de/internal/storage/storageutil/control_client.go#L30 rather than defining it here.
It will give rise to maintenance issues when this is changed.